src/azure-cli/azure/cli/command_modules/sqlvm/_params.py

```diff
@@ -363,3 +363,7 @@ def load_arguments(self, _):
                    help='Name of the Log Analytics workspace to associate with VM.')
         c.argument('workspace_rg',
                    help='Resource group containing the Log Analytics workspace.')
+        c.argument('workspace_sub',
+                   help='Subscription containing the Log Analytics workspace.')
+        c.argument('agent_rg',
+                   help='Resource group where we provision the AMA Agent resources.')
```

Comment:
Are these two parameters required? Adding required argument is breaking change which we need to pay much attention to

Reply:
both are made optional now.